Makefile

```diff
@@ -1,9 +1,14 @@
+RUNC_IMAGE=runc_dev
```

Comment:
Where does the runc_test image come from? How does it get the go environment? Is it coming from this repo: https://hub.docker.com/_/golang/?